.gitlab-ci.yml

```diff
@@ -53,7 +58,7 @@ stages:
     include:
       - local: '.gitlab/custom-jobs-and-variables.yml'
       - project: 'radiuss/radiuss-shared-ci'
-        ref: v2022.12.0
+        ref: v2023.03.0rc
```

Comment:
Do these need to be resynced with some changes I recall seeing come in this morning from a branch the @adrienbernede is working on?

Reply:
For now it’s good to go. I’m working on the updating the specs in the CI, and that will eventually go in release v2023.03.0 of radiuss-shared-ci.